src/components/sections/employeeHighlight/employeeHighlight.module.css

```diff
@@ -0,0 +1,69 @@
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 0 2rem;
+  max-width: 64rem;
+  margin: 5rem auto;
+  gap: 1.5rem;
+  color: var(--text-primary);
+
+  @media (max-width: 750px) {
```

Comment:
The breakpoint for tablet is 834px and 425px for mobile 📱

Reply:
Ole Petter and I decided on 750px because this happens when you go past 750px 😅 
<img width="720" alt="Screenshot 2024-12-05 at 13 40 57" src="https://github.com/user-attachments/assets/7f277d2b-7646-427c-9e8c-33d364541348">

Reply:
But that will obviously depend on the length of the names 😢

Reply:
We also should rework our typography a bit, because the font-size should decrease with screensize, I do think that might help in this case actually!

Reply:
> We also should rework our typography a bit, because the font-size should decrease with screensize, I do think that might help in this case actually!

Ah, very true!

Reply:
And yep different length of different names is always a pain 😅

Reply:
Done in https://github.com/varianter/variant.no/pull/943/commits/d62bd413c64ab6ff78a073ea65c58096b0460eb5